src/analytics.ts

```diff
@@ -0,0 +1,19 @@
+import { machineId } from 'node-machine-id';
+import RudderAnalytics from "@rudderstack/rudder-sdk-node"
+
+let client: RudderAnalytics | undefined;
+try {
+    client = new RudderAnalytics(process.env.RUDDER_STACK_KEY!, {
```

Comment:
how will this work when people install the CLI? they will not create and env file 🤔